api/src/main/java/org/pmiops/workbench/profile/ProfileService.java

```diff
@@ -172,12 +169,6 @@ public void validateInstitutionalAffiliation(Profile profile) {
   public void updateProfileForUser(DbUser user, Profile updatedProfile, Profile previousProfile) {
     validateUpdatedProfile(updatedProfile, previousProfile);
 
-    if (!userProvider.get().getGivenName().equalsIgnoreCase(updatedProfile.getGivenName())
```

Comment:
moved to validateUpdatedProfile() because it also applies to the adminUpdateProfile()